_schemaV2.graphql

```diff
@@ -16126,6 +16210,31 @@ type Viewer {
     size: Int
     slugs: [String!]
   ): [MarketingCollection]
+  match(
```

Comment:
What's the use-case for both a list `match` and also `matchConnection`, is `matchConnection` sufficient?

Reply:
Yeah I'll delete `match`